src/test/java/com/flipt/api/FliptContainer.java

```diff
@@ -0,0 +1,40 @@
+package com.flipt.api;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FliptContainer extends GenericContainer<FliptContainer> {
+
+    private static final DockerImageName IMAGE_NAME = DockerImageName.parse("flipt/flipt").withTag("latest");
+    private static final String DROP_TARGET_PATH = "/tmp/drop.yaml";
+    private static final int PORT = 8080;
+
+    public FliptContainer() {
+        super(IMAGE_NAME);
+        addExposedPort(PORT);
+    }
+
+    public void importConfiguration(Path configuration) throws Exception {
+        var targetPath = "/tmp/%s".formatted(configuration.getFileName());
+
+        copyFileToContainer(Transferable.of(Files.readAllBytes(configuration)), targetPath);
+        this.execInContainer("./flipt", "import", targetPath);
```

Comment:
```suggestion
        this.execInContainer("./flipt", "import", "--drop", targetPath);
```
we could prob just drop before import, then we could get rid of the `afterEach` and `dropData` all together I think

Reply:
Yes, that would definitely work.

My reasoning was that I was trying to avoid unexpected behavior as much as possible.
So I separated the dropping and importing into separate methods.
If you'd prefer to do everything in one method, I would at least rename the method to `dropAndImportConfiguration`.

Another way to clean up the state between each test method is to make the container in the test class non-static.
This way, testcontainers will restart it for each test. This may have runtime implications. However, a local test with the current test code was still fast.

wdyt?

Reply:
yeah, i think another benefit of running drop before the tests is that the data still persists in case of test failure to allow you to see the state when the test failed. this might help when running locally at least or when creating new tests. wdyt?